spec/models/category_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+RSpec.describe Category, type: :model do
+  describe 'test description' do
+    before :all do
+      @user = User.first
+      @category = Category.new(user: @user, name: 'MC donald')
+    end
+    it 'expect category to be saved' do
+      expect(@category.save).to be_truthy
+    end
+    it 'expect category to be found' do
+      category = Category.create(user: @user, name: 'MC donald')
+      expect(Category.find(category.id)).to eql(category)
+    end
+    it 'expect Category to fail saving without user' do
+      category = Category.new(name: 'MC donald')
+      expect(category.save).to be_falsey
+    end
+  end
+end
```

Comment:
- [ ] Four tests fail after running `rspec spec` command. Let's try to fix them so that your project is complete:

<img width="533" alt="image" src="https://user-images.githubusercontent.com/77190203/189213937-86c920d5-d6f1-44af-846f-311ee6242e8e.png">

Reply:
Dear @medaminedev66. The issue with the failing test is related to the seed data. As I have mentioned in my pull request. some tests are dependent on the seed file. but if it's a bad practice to use seed data to run test the working of the app. ill make sure to ignore data-related tests. I'll wait for your response on how to proceed on this.